src/Item.js

```diff
@@ -114,37 +140,42 @@ function Item({ userToken, item, status }) {
   };
 
   return (
-    <li>
+    <ItemStyle>
       <label htmlFor={itemName} aria-label={itemName + ' (' + status + ')'}>
-        <input
+        <Checkbox
           type="checkbox"
           name={itemName}
           id={itemName}
           checked={checked}
           onChange={handleClick}
+          color="secondary"
         />
         {itemName}
       </label>
+
       {/* // [delete-item] 1. Create a button next to each item in list */}
-      <button type="button" onClick={handleOpenModal}>
-        X
-      </button>
+      <DeleteBtn type="button" onClick={handleOpenModal}>
```

Comment:
@ablizben if you add an aria-label to this it will fix an a11y error:

```suggestion
      <DeleteBtn type="button" onClick={handleOpenModal} aria-label={`delete ${itemName}`}>
```

Reply:
Thanks, Lauren!